modules/delegates.js

```diff
@@ -220,26 +220,12 @@ __private.forge = function (cb) {
 		}
 
 		library.sequence.add(function (cb) {
-			async.series({
-				getPeers: function (seriesCb) {
-					return modules.transport.getPeers({limit: constants.maxPeers}, seriesCb);
-				},
-				checkBroadhash: function (seriesCb) {
-					var consensus = modules.peers.getConsensus();
-					if (modules.transport.poorConsensus(consensus)) {
-						return setImmediate(seriesCb, ['Inadequate broadhash consensus', consensus, '%'].join(' '));
-					} else {
-						return setImmediate(seriesCb);
-					}
-				}
-			}, function (err) {
-				if (err) {
-					library.logger.warn(err);
-					return setImmediate(cb, err);
-				} else {
-					return modules.blocks.process.generateBlock(currentBlockData.keypair, currentBlockData.time, cb);
-				}
-			});
+			var consensus = modules.peers.getConsensus();
+			if (modules.transport.poorConsensus(consensus)) {
+				return setImmediate(cb, ['Inadequate broadhash consensus', consensus, '%'].join(' '));
+			}
+			library.logger.info(['Broadhash consensus now', self.consensus, '%'].join(' '));
```

Comment:
There is no `self.consensus` (should be just `consensus`)

Reply:
Move that log inside `modules.peers.getConsensus()`, which is proper place.

Reply:
I don't think that moving the logging part into `getConsensus` function is a good idea. It's against the Single Responsibility Principle.